pycognaize/tests/test_document/test_field/test_table_field.py

```diff
@@ -65,8 +71,28 @@ def setUp(self):
         page_n = self.raw_table_with_group_key['tags'][0]['page']
         self.pages = {page_n: create_dummy_page(page_n=page_n,
                                                 path=self.snap_storage_path)}
-        self.tbl_field = TableField.construct_from_raw(
-            self.raw_table_with_group_key, pages=self.pages)
+
+        self.html = HTML(path=os.path.join(RESOURCE_FOLDER, 'snapshots'),
+                         document_id=self.doc_id2)
+
+        with open(os.path.join(RESOURCE_FOLDER, 'snapshots', self.doc_id2,
+                               'document.json')) as document_json:
+            self.data = json.load(document_json)
+
+        self.tbl_field = TableField.construct_from_raw(self.raw_table_with_group_key,
+                                                       pages=self.pages)
+        table_field = self.data['input_fields']['table'][0]
+        self.raw_table_tag = table_field[IqFieldKeyEnum.tags.value][0]
+        self.tbl_tag = HTMLTableTag.construct_from_raw(raw=self.raw_table_tag,
+                                                       html=self.html)
+        self.tbl_field_2 = TableField(name="", tag=self.tbl_tag)
+
+    def test_get_table_title(self):
+        title_1 = self.tbl_field_2.get_table_title(n_lines_above=15, margin=9)
+        self.assertEqual(title_1, '\u200b (exact name of registrant as specified in its charter):usa truck inc.')
```

Comment:
Please strip \\u200b from the output and from the expected string and check assert equals